src/components/dashboard/employees/CreateEmployee.js

```diff
@@ -87,15 +86,6 @@ class CreateEmployee extends Component {
       this.setState({ firstNameError: '' });
     }
 
-    // Validate workEmail
-    const workEmail = this.state.workEmail;
```

Comment:
Should we require workEmail if it's an existing employee?

Reply:
this form is for new hires only.  but on the swagprofile for existing we should probably require it huh?

Reply:
Yeah let's require it for existing employees..